data/helpers.d/multimedia

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Install or update the main directory yunohost.multimedia
+#
+# usage: ynh_multimedia_build_main_dir
+ynh_multimedia_build_main_dir () {
+        local ynh_media_release="v1.2"
+        local checksum="806a827ba1902d6911095602a9221181"
+
+        # Download yunohost.multimedia scripts
+        wget -nv https://github.com/YunoHost-Apps/yunohost.multimedia/archive/${ynh_media_release}.tar.gz 
```

Comment:
I too am wondering if this shouldn't be included in the core ?... Creating a new repo + all this release and checksum stuff looks like a lot of logistic for only 2 shell scripts (also what about the updates later).

Can't we just add those in the core somehow, and add a call to `ynh_media_build.sh` during the postinstall if there isn't any significant overhead in running it ?

Reply:
@alexAubin, as said, I can integrate that helper into the core, if you or someone else show me where I should do that.